x-pack/plugins/alerting/server/lib/wrap_scoped_cluster_client.ts

```diff
@@ -152,11 +154,10 @@ function getWrappedTransportRequestFn(opts: WrapEsClientOpts) {
     }
 
     // No wrap
-    return (await originalRequestFn.call(
-      opts.esClient.transport,
-      params,
-      options
-    )) as Promise<TResponse>;
+    return (await originalRequestFn.call(opts.esClient.transport, params, {
+      requestTimeout: opts.requestTimeout,
+      ...options,
```

Comment:
same comment as above

Reply:
Resolved in this commit [a71807b](https://github.com/elastic/kibana/pull/175484/commits/a71807bcd872ae278c19718fe48d8d64864a6e53)